events.html

```diff
@@ -57,21 +57,68 @@
 
   <body>
     <nav>
-      <ul class="navbar-ul">
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="nav-login">
+        <a
+          class="btn-login"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button class="btn-login-text">
+            Sign In
+            <img
+              src="img/github.png"
+              alt="GitHub Icon"
+              height="15px"
+              width="15px"
+            />
+          </button>
+        </a>
+        <div class="user-greet">
+          <div class="user-greet-msg">Hello text</div>
```

Comment:
Why are we mentioning `text` here?

Reply:
It seems to be a placeholder text I think

Reply:
could you confirm once please, rest is good

Reply:
Yes, it is actually a placeholder, we can rename to `Hello User`, which makes more sense.